otx/api/usecases/exportable_code/demo/demo_package/executors/asynchronous.py

```diff
@@ -39,12 +39,13 @@ def run(self, input_stream: Union[int, str], loop: bool = False) -> None:
         next_frame_id_to_show = 0
         stop_visualization = False
 
-        for frame in streamer:
+        for (frame, input_path) in streamer:
```

Comment:
I think it can raise an error if streamer is `ThreadStreamer`.

Reply:
Sure, it will be much easier. But it would cause a problem, that input and output names aren't consistent and it's hard to identify, which image was the source, based on naming

But if additional output in streamer is highly unwelcome, I can write a standalone function to get input names from args.input, without changing the streamer code.